idd/Energy+.idd.in

```diff
@@ -42754,6 +42779,100 @@ Coil:Heating:DX:VariableRefrigerantFlow,
+       \note If this field is left blank, the unit is available the entire simulation
+  A3,  \field Coil Air Inlet Node
+       \required-field
+       \type node
+       \note the inlet node to the coil
+  A4,  \field Coil Air Outlet Node
+       \required-field
+       \type node
+       \note the outlet node to the coil
+  N1,  \field Rated Total Cooling Capacity
+       \required-field
+       \autosizable
+       \type real
+       \units W
+       \minimum> 0.0
+       \note Supply air fan heat is not included
+  N2,  \field Rated Sensible Heat Ratio
+       \required-field
+       \autosizable
+       \type real
+       \minimum> 0.0
+       \note Supply air fan heat is not included
+  N3,  \field Indoor Unit Reference Superheating 
+       \type real
+       \units deltaC 
+       \minimum 0.0
+       \default 5.0
+  A5,  \field Indoor Unit Evaporating Temperature Function of Superheating Curve Name    
+       \required-field   
+       \type object-list
+       \object-list QuadraticCurves
+       \object-list UniVariateTables
+  A6;  \field Name of Water Storage Tank for Condensate Collection
+       \type object-list
+       \object-list WaterStorageTankNames
+ 
+Coil:Heating:DX:VariableRefrigerantFlow:FluidTemperatureControl,
+       \memo This is a key object in the new physics based VRF model applicable for Fluid
+       \memo Temperature Control. It describes the the indoor unit coil of the system at heating mode.
+       \memo Used with ZoneHVAC:TerminalUnit:VariableRefrigerantFlow. 
+       \memo Outdoor unit is modeled separately, see AirConditioner:VariableRefrigerantFlow:FluidTemperatureControl.
+       \min-fields 5
+  A1,  \field Name
+       \required-field
+       \type alpha
+       \reference HeatingCoilsDXVarRefrigFlowFluidTemperatureControl
+  A2,  \field Availability Schedule
+       \type object-list
+       \object-list ScheduleNames
+       \note Enter the name of a schedule that defines the availability of the coil
+       \note Schedule values of 0 denote the unit is off. All other values denote the unit is available
+       \note If this field is left blank, the unit is available the entire simulation
+  A3,  \field Coil Air Inlet Node
+       \required-field
+       \type node
+       \note the inlet node to the coil
+  A4,  \field Coil Air Outlet Node
+       \required-field
+       \type node
+       \note the outlet node to the coil
+  N1,  \field Rated Total Heating Capacity
+       \required-field
+       \autosizable
+       \type real
+       \units W
+       \minimum> 0.0
+       \autosizable
+       \note Supply air fan heat is not included
+  N2,  \field Indoor Unit Reference Subcooling 
+       \type real 
+       \units deltaC
+       \minimum 0.0
+       \default 5.0
+  A5;  \field Indoor Unit Condensing Temperature Function of Subcooling Curve Name    
+       \required-field   
+       \type object-list
+       \object-list QuadraticCurves	  
```

Comment:
@rongpengzhang IDF Editor doesn't like the trailing spaces on these \object-list references. I only found that on these two, but there's one somewhere else that IDF Editor is complaining about.